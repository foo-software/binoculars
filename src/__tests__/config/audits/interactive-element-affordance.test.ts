import auditInteractiveElementAffordance from '../../../config/audits/interactive-element-affordance';

describe('interactive-element-affordance audit', () => {
  it('should match snapshot', () => {
    expect(auditInteractiveElementAffordance('en')).toMatchSnapshot();
  });
});
