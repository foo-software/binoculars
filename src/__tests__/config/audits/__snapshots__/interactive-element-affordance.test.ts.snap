// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`interactive-element-affordance audit should match snapshot 1`] = `
Object {
  "description": "Interactive elements, such as links and buttons, should indicate their state and be distinguishable from non-interactive elements. [Learn more](https://web.dev/interactive-element-affordance/).",
  "title": "Interactive elements indicate their purpose and state",
}
`;
